apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -268,18 +245,17 @@ export class SpecialAnnouncementActionBlock extends Component {
       <TwoColumnActionBlock
         id="teacher-nomination-announcement"
         imageUrl={pegasus(
-          '/shared/images/fill-540x289/teacher-announcement/professional-learning-2019-trophy.jpg'
+          '/shared/images/fill-540x289/teacher-announcement/professional-learning-2019-3.jpg'
         )}
-        imageExtra={imageExtra}
-        teacherStyle={true}
-        subHeading={i18n.specialAnnouncementHeadingJoinProfessionalLearning2019Trophy()}
-        subHeadingSmallFont={false}
-        description={i18n.specialAnnouncementDescriptionJoinProfessionalLearning2019Trophy()}
+        subHeading={i18n.specialAnnouncementHeadingJoinProfessionalLearning2019()}
+        subHeadingSmallFont={true}
+        description={i18n.specialAnnouncementDescriptionJoinProfessionalLearning2019()}
+        imageExtra={false}
+        teacherStyle={false}
         buttons={[
           {
-            id: 'teacher-nomination-button',
```

Comment:
Should we keep an id for this **Join us** button so we can do analysis in GA later? It could be useful if there are more than 1 way teacher can get to application form